java/arcs/android/storage/handle/AndroidHandleManager.kt

```diff
@@ -25,6 +25,7 @@ import kotlin.coroutines.EmptyCoroutineContext
  * [ActivationFactory] with one that generates [ServiceStore] instances that can
  * communication with a running [StorageService].
  */
+@Deprecated("Use arcs.core.host.EntityHandleManager")
```

Comment:
Optional, but it would be really great if you could file a GitHub issue for this refactor, and for all of these TODOs and `@Deprecated`s, include the issue number, e.g. `#1234`. It should help make tracking them all down much easier.

Reply:
Will do. I already started tracking the TODOs here: b/152434568. I'll add TODOs in the relevant spots in the code here.